cluster/gce/trusty/kube-manifest/fluentd-es.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
```

Comment:
Why we need an extra copy here for trusty? Shouldn't they are same for all os distro?